### PR TITLE
4 get api articles

### PR DESCRIPTION
**Pull request by Geo-Beard**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -15,8 +17,25 @@ app.use("*", (req, res) => {
...
+    res.status(400).send({ message: "400 - Bad request" });
+  }
+  next(err);
+});
+
 //Custom errors
 
+app.use((err, req, res, next) => {
+  if (err.status && err.message) {
+    console.log(err.status);
```
Comment (two-triangles) [inline app.js:31]: dont forget to remove console.logs once you have completed a task :)

File: app.js [hunk 2]
```diff
@@ -15,8 +17,25 @@ app.use("*", (req, res) => {
...
+    console.log(err.message);
+    res.status(err.status).send({ message: `${err.message}` });
```
Comment (two-triangles) [inline app.js:33]: I think you can just pass in err.message as it's already a string :)

File: __tests__/app.test.js [hunk 3]
```diff
@@ -29,11 +29,54 @@ describe("GET /api/topics", () => {
...
+        });
+    });
+  });
+  describe("SAD PATHS", () => {
+    test("responds with 404 if article_id does not exist", () => {
+      return request(app)
+        .get("/api/articles/42")
+        .expect(404)
+        .then(({ body: { message } }) => {
+          console.log(message);
```
Comment (two-triangles) [inline __tests__/app.test.js:69]: again just make sure you are removing console.logs as you go

File: app.js (hunk 1 above)
Comment (Geo-Beard) [reply app.js:31]: Yep! I've been through and removed them - but I think I've removed them on the next branch.

**Comment by two-triangles** [comment]
Looking good so far Simon! 

 - watch out for console.logs that don't want to be there 
 - have a think about refactoring your errors into their own directory at some point - will make the app.js much clearer

**Pull request merged**